code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -304,6 +343,92 @@
 		busy = 0
 		stop_automated_movement = 0
 
+/mob/living/simple_animal/hostile/giant_spider/verb/web()
+	set name = "Spin Web"
+	set desc = "Create a web that slows down movement."
+	set category = "Greimorian"
+
+	var/obj/effect/spider/stickyweb/W = locate() in get_turf(src)
+	if(!W)
+		to_chat(usr, SPAN_NOTICE("\The [src] begins to secrete a sticky substance."))
+		if(!do_after(src, 20)) return
+		new /obj/effect/spider/stickyweb(src.loc)
```

Comment:
You should probably get the turf of the location with `get_turf`, the loc otherwise could be a container, nullspace, another mob, or anything else that probably should not have a web attached

Reply:
Handled